### PR TITLE
Add "Desired" to base compensation

### PR DESCRIPTION
**Pull request by pixelwake**
This was raised by one of the SME's during training. The goal is to make it clear that this isn't based on what you have been making but what you would like to make. Increase clarity to prevent applicants from limiting their level due to their current compensation only.

### CODE REVIEW

File: pages/pay-grades.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This information is intended as a guideline only. Use this information to help t
 
 |               | **Grade 13**      |**Grade 14**|**Grade 15**|
 | -----------------:|:-------------:| :-----:| :-----:|
-| **Base Compensation**     | $73,115 to $95,048 | $86,399 to  $112,319 |$101,630 to $132,122|
+| **Desired Base Compensation**     | $73,115 to $95,048 | $86,399 to  $112,319 |$101,630 to $132,122|
```
Comment (fureigh) [inline pages/pay-grades.md:19]: @rooneywp I appreciate the rationale behind this change, but I'm hoping there's a different place on this page where the "desired" distinction could be made instead. Technically these are the base compensations (period) that are associated with their respective grades, but it's the grade level (with its associated compensation) that's desired, no?

Comment (pixelwake) [inline pages/pay-grades.md:19]: @fureigh I see your point and I think it raises some more general confusion that leads me to think that we should pull salary away from these other items. 

The other items in the table describe the level of experience the applicant needs to be qualified for that level. Putting Salary next to them makes me feel like it's saying that those are the salary's that you should have to qualified for that level. I think that's the issue I was having/trying to address by calling out desired there.

Comment (pixelwake) [inline pages/pay-grades.md:19]: @fureigh I was experimenting a bit with different formatting and couldn't find something that I liked. Maybe we are over engineering this and we should consider user data? Applicants have been applying and haven't contacted our team with any questions about grade level or anything. Maybe from the applicants perspective it's working as intended. We could interview some of the new applicants about their experience?

**Comment by pixelwake** [comment]
Closing this PR, no additional feedback has been provided by users. This will ultimately be addressed if we create the wizard tool.

**Pull request closed without merging**